src/store/app.ts

```diff
@@ -1,13 +1,15 @@
 import { atom } from 'jotai';
 import { atomWithStorage } from 'jotai/utils';
-import type { Amount, Brand, DisplayInfo } from '@agoric/ertp';
-import type { PursesJSONState } from '@agoric/wallet-backend';
-import type { Ratio } from '@agoric/zoe/src/contractSupport';
-
 import { makeDisplayFunctions } from 'utils/displayFunctions';
 import { mapAtom } from 'utils/helpers';
 import { networkConfigs } from 'config';
 
+// Ambient
+import '@agoric/ertp/src/types';
+import '@agoric/zoe/src/contractSupport';
+import '@agoric/wallet-backend/src/types';
+import { PursesJSONState } from '@agoric/wallet-backend';
```

Comment:
Does this have to be imported after the types?

Reply:
Not sure how line 11 got there. Maybe It should go back up to where it was as `import type { PursesJSONState } from '@agoric/wallet-backend';`